framework/auth/core.py

```diff
@@ -57,6 +57,22 @@ def validate_history_item(item):
     startYear = item.get('startYear')
     endMonth = item.get('endMonth')
     endYear = item.get('endYear')
+
+    if startYear:
+        try:
+            int(startYear)
+        except ValueError:
+            raise ValidationValueError('Please enter a valid year.')
+
+    if endYear:
+        try:
+            int(endYear)
+        except ValueError:
+            raise ValidationValueError('Please enter a valid year.')
+
+    if int(startYear)/10000 == 0 or int(endYear)/10000 == 0:
```

Comment:
I don't think this validation logic is correct. Why check if floor division with `10000` equals 0?